app/assets/javascripts/messaging.js

```diff
@@ -4,16 +4,30 @@
   console.log("inside angular")
   app.controller('ListOfMessagesController', ['$scope', '$http', '$routeParams', '$timeout', '$interval', function($scope, $http, $routeParams, $timeout, $interval){
      //var Id = $routeParams.id - 1; //Id to make a difference with id
+     $scope.loading = true; //Spinner with angular starting, not working
 
      function updateMessages(){
+       //$scope.loading = false; //Spinner with angular sttoping, not working
+
       return $http.get("/messages.json")
         .then(function(response){
+
           $scope.messages = response.data
+          })
+        .finally(function(){
+          $scope.loading = false; //Spinner with angular sttoping, not working
+
```

Comment:
Adding this `.finally` doesn't stop the never stopping spinner either. Not in the right place? Not using the right controller? wasting time using this type of spinner here? enough questions?